tests/vibrato_tests.ts

```diff
@@ -103,8 +104,8 @@ function withBend(options: TestOptions, contextBuilder: ContextBuilder): void {
       positions: [{ str: 2, fret: 10 }],
       duration: 'q',
     })
-      .addModifier(0, new Bend('Full', false))
-      .addModifier(0, new Vibrato().setVibratoWidth(60)),
+      .addModifier(0, new Vibrato().setVibratoWidth(60))
+      .addModifier(0, new Bend('Full', false)),
```

Comment:
Does the call order have any effect? If not it seems that vibrato_tests.ts has no change and should stay as it was.

Reply:
Changed it back.   Order doesn't matter.